src/components/v2/V2Project/V2ManageTokensSection/V2RedeemModal.tsx

```diff
@@ -192,80 +225,53 @@ export default function V2RedeemModal({
                   {formatWad(totalBalance ?? 0, { precision: 0 })}{' '}
                   {tokensTextShort}
                 </div>
-                <div
-                  style={{
-                    cursor: 'default',
-                    fontSize: '0.8rem',
-                    fontWeight: 500,
-                    color: colors.text.tertiary,
-                  }}
-                >
-                  ({share}% of supply)
-                </div>
               </div>
             </Trans>
           </p>
           <p style={statsStyle}>
             <Trans>
-              Currently worth:{' '}
+              Redemption value:{' '}
               <span>
                 <CurrencySymbol currency="ETH" />
                 {formatWad(maxClaimable, { precision: 4 })}
```

Comment:
**Issue**: Think we need to make this the same precision as the redeem input `extra` (which is to 8dp at `RedeemModal:232`)

Reply:
@johnnyd-eth yes v good pickup.

I did 2 things:
- improved ETHAmount some more. For low values, the tooltip on the `~0` ETH amount shows an 8-precision formatted amount.
- Used ETHAmount in the "you will receive" text.

| super low | normal |
| --- | --- |
| <img width="571" alt="Screen Shot 2022-08-03 at 10 23 39 PM" src="https://user-images.githubusercontent.com/12551741/182607642-4384b2ef-4d09-4177-9893-cee1e566f1a1.png"> | <img width="562" alt="Screen Shot 2022-08-03 at 10 23 46 PM" src="https://user-images.githubusercontent.com/12551741/182607685-d65bc92a-d4f4-41ce-99c2-90685e0b1f61.png"> |